zerver/tests/test_push_notifications.py

```diff
@@ -923,13 +925,23 @@ def test_send_remove_notifications_to_bouncer(self) -> None:
 
         with self.settings(PUSH_NOTIFICATION_BOUNCER_URL=True), \
                 mock.patch('zerver.lib.push_notifications'
-                           '.send_notifications_to_bouncer') as mock_send_android, \
+                           '.send_notifications_to_bouncer') as mock_send, \
                 mock.patch('zerver.lib.push_notifications.get_base_payload',
                            return_value={'gcm': True}):
```

Comment:
Can we remove the mocking of `get_base_payload` in a prep commit, and just call the real `get_base_payload`?  I think it makes the below tests a lot more confusing to not have the actual payload there, but instead this `gcm: True` thing, especially as we're adding the Apple case..